content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,74 +1,378 @@
 ==================
-Avatax integration
+AvaTax integration
 ==================
 
-Avatax is a tax calculation provider that can be integrated in Odoo.
+Avalara's *AvaTax* is a cloud-based tax software. Integrating AvaTax with Odoo provides real-time
+and region-specific tax calculations when users sell, purchase, and invoice items in Odoo. *AvaTax*
+tax calculation is supported in every United Nations charted country, including inter-border
+transactions.
 
-.. _avatax/credentials:
+.. important::
+   AvaTax is only available for databases/companies with locations in the United States and Canada.
+
+*AvaTax* accounts for location-based tax rates for each state, county and city. It improves
+remittance accuracy by paying close attention to laws, rules, jurisdiction boundaries, and special
+circumstances like tax holidays and product exemptions. Companies who integrate with *AvaTax* can
+maintain control of tax-calculations in-house with this simple :abbr:`API (application programming
+interface)` integration.
+
+Avalara provides the following services:
+
+- AvaTax (sales and use tax calculations)
+- Avalara returns (prepare, file, and remit sales tax returns)
+- Avalara Exemption Certificate Management (collect, store, and manage documents)
+- Avalara tax Research (Get tax research in plain language)
+- Avalara Business License Services (project based assistance to help confront challenges to their
+  license operations)
+
+.. important::
+   Some limitations exist in Odoo while using AvaTax for tax calculation:
+
+   - AvaTax is not supported in Odoo *Point of Sale*, because a dynamic tax calculation model is
+     excessive for transactions within a single delivery address, such as stores or restaurants.
+
+   - AvaTax and Odoo use the company address and not the warehouse address.
```

Comment:
No need for a blank line between bullet points of the same level :)
```suggestion
   - AvaTax is not supported in Odoo *Point of Sale*, because a dynamic tax calculation model is
     excessive for transactions within a single delivery address, such as stores or restaurants.
   - AvaTax and Odoo use the company address and not the warehouse address.
```